tools/website.sh

```diff
@@ -2,6 +2,6 @@
 
 NODE_VER=`node -v`
 
-if [[ $NODE_VER =~ ^v6 && $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then
+if [[ $NODE_VER =~ ^v6 && $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "main" ]]; then
```

Comment:
How can this work? We are no longer using travis 🤔

Reply:
cc @lmammino

Reply:
I think this should be removed since we are not relying on travis

Reply:
If we are not using travis, we should have the same script in github actions... double checking now...

Reply:
Yes, I can confirm we do the call here: https://github.com/fastify/fastify/blob/cceac26664b204e6c196e48dc156a0fd761268c2/.github/workflows/website.yml#L16

Also, this file has been already updated as part of this PR using the correct branch as a trigger for the pipeline. I think the PR is still good.

I have no idea about whether we should remove this specific file or not. Are we using it somewhere else? It is definitely not used in the new GitHub actions workflow linked above, so I wouldn't expect to see this used somewhere else...

Reply:
I think we can leave it for now

Reply:
Could you just drop it as part of this PR?

Reply:
@mcollina  Are you talking about removing `website.sh` completely?

Reply:
Yes exactly

Reply:
Done